### PR TITLE
Needinfo assignees in bugs filed by fuzzing that have a bisection to ask them to set the Regressed By field

### LINKED ISSUE #1039: Needinfo assignees in bugs filed by fuzzing that have a bisection to ask them to set the Regressed By field
The bisection comment currently looks like this: https://bugzilla.mozilla.org/show_bug.cgi?id=1683535#c6.
It used to look like this: https://bugzilla.mozilla.org/show_bug.cgi?id=1619324#c3.

The main difficulty is getting all fuzzing bugs.
They are all blocking some fuzzing meta bug, the fuzzing meta bugs are in turn blocking https://bugzilla.mozilla.org/show_bug.cgi?id=fuzz, but there might be levels of indirection between them (a meta blocking another meta, and then blocking `fuzz`).

### PR DESCRIPTION
**Pull request by marco-c**
Fixes #1039

Getting blocking bugs recursively appears to be pretty slow.

### CODE REVIEW

File: auto_nag/scripts/fuzzing_bisection_without_regressed_by.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            "v1": ",".join(str(bid) for bid in blocked_ids),
+            "f2": "regressed_by",
+            "o2": "isempty",
+            "n3": 1,
+            "f3": "regressed_by",
+            "o3": "everchanged",
+            "n4": 1,
+            "f4": "longdesc",
+            "o4": "casesubstring",
+            "v4": "since this bug contains a bisection range, could you fill (if possible) the regressed_by field",
```
Comment (calixteman) [inline auto_nag/scripts/fuzzing_bisection_without_regressed_by.py:50]: I don't know if it's possible, but it'd be nice to get this string from a configuration file from the tool which creates it.

File: auto_nag/scripts/fuzzing_bisection_without_regressed_by.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+    def filter_bugs(self, bugs):
+        # Exclude meta bugs.
+        bugs = {bug["id"]: bug for bug in bugs.values() if not bug["is_meta"]}
+
+        # Exclude bugs assigned to nobody.
+        bugs = {
+            bug["id"]: bug
+            for bug in bugs.values()
+            if bug["assigned_to"] != "nobody@mozilla.org"
```
Comment (calixteman) [inline auto_nag/scripts/fuzzing_bisection_without_regressed_by.py:106]: nit: https://github.com/mozilla/relman-auto-nag/blob/master/auto_nag/utils.py#L123

File: templates/fuzzing_bisection_without_regressed_by_needinfo.txt [hunk 3]
```diff
@@ -0,0 +1,2 @@
+:{{ nickname }}, since this bug contains a bisection range, could you fill (if possible) the regressed_by field?
+{{ documentation }}
```
Comment (calixteman) [inline templates/fuzzing_bisection_without_regressed_by_needinfo.txt:2]: Don't forget: https://wiki.mozilla.org/Release_Management/autonag

**Review by calixteman — APPROVED** [review]
Just ping me once you merge it to update the vm.

File: auto_nag/scripts/fuzzing_bisection_without_regressed_by.py (hunk 2 above)
Comment (marco-c) [reply auto_nag/scripts/fuzzing_bisection_without_regressed_by.py:106]: Done in 846b827a69fdd86dc5a6ab08e43100e0b40bce32.

File: templates/fuzzing_bisection_without_regressed_by_needinfo.txt (hunk 3 above)
Comment (marco-c) [reply templates/fuzzing_bisection_without_regressed_by_needinfo.txt:2]: Done https://wiki.mozilla.org/Release_Management/autonag#Fuzzing_bisection_without_regressed_by.

**Pull request merged**